extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/databaseSettingsPage.ts

```diff
@@ -128,6 +135,31 @@ export class DatabaseSettingsPage extends BasePage {
 		this._endIpAddressTextRow = this.wizard.createFormRowComponent(view, constants.EndIpAddressLabel, '', this._endIpAddressTextbox, true);
 	}
 
+	private createFirewallToggle(view: azdata.ModelView) {
+		this._firewallToggleDropdown = view.modelBuilder.dropDown().withProps({
```

Comment:
Did you talk with the designers about using a dropdown for this? Seems like a checkbox would be cleaner.